data/Localization/german_(germany)/global.ini

```diff
@@ -8513,8 +8560,8 @@ Hints_Melee_KnifeLight=Um einen schnellen Hieb auszuführen, tippe auf ~action(p
 Hints_Melee_LightAttacks=Um einen schnellen Jab zu werfen, tippe auf ~action(player|melee_AttackLightLeft) oder ~action(player|melee_AttackLightRight).<left><right></right></left>
 Hints_Melee_Stun=Wenn du von Fäusten getroffen wirst, erleidest du Betäubungsschaden. Dieser ist nicht tödlich, aber wenn du zu oft getroffen wirst, wirst du niedergeschlagen.
 Hints_Melee_TakedownFists=Drücken Sie ~action(player|weapon_melee) für einen nicht-tödlichen Takedown.
-Hints_Melee_TakedownGun=Press ~action(player|weapon_melee) <takedown gun="">for a non-lethal takedown.</takedown>
-Hints_Melee_TakedownWeapon=Press ~action(player|weapon_melee)<takedown melee="">&nbsp;for a lethal takedown.</takedown>
+Hints_Melee_TakedownGun=Drücken Sie ~action(player|weapon_melee) <takedown gun="">für eine nicht-tödliche Überwältigung.</takedown>
+Hints_Melee_TakedownWeapon=Drücken Sie ~action(player|weapon_melee)<takedown melee="">&nbsp;für eine tödliche Überwältigung.</takedown>
```

Comment:
I would replace the "Drücken Sie" with a "Drücke" (and the line above it)
and maybe find another word for "Überwältigung". I would maybe just leave it by "takedown"